### PR TITLE
Feat/fileplayer

### PR DESCRIPTION
**Pull request by charjac**
in main package
 - add the ability to play a local file by passing its path as arg, and add mimeType association
```
nuclear myfile.mp3
```
 - refactor window event sender
 - add Tray service

### CODE REVIEW

**Comment by charjac** [comment]
I was also thinking about drag and drop on the queue to add tracks, what do you think ?

**Comment by nukeop** [comment]
As in, dragging files from the outside into the player? That's a really good idea.

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "electron": "^4.2.11",
-    "electron-builder": "^21.2.0",
+    "electron-builder": "^20.44.4",
```
Comment (nukeop) [inline package.json]: why the downgrade?

Comment (charjac) [reply package.json]: better error log when trying to build locally, i can rollback this now

**Comment by charjac** [comment]
I had a few things: 
- dragging files and/or folders in the queue add tracks to queue and their folder and metadata to the store
- minimal design ('-_-) for the queue when dragHover with files
- redirect on library page when a file in run at startup
- prevent multiple instance of electron being run

**Comment by charjac** [comment]
For me its ok

**Comment by nukeop** [comment]
Looks good to me, fantastic work as always.

**Comment by nukeop** [comment]
I'll change a few things (droppable box styles and state added via HOC not class component) though

**Comment by nukeop** [comment]
I also found a bug where dragging the same song twice causes the second copy to not load the source correctly

**Comment by charjac** [comment]
ok i'll check this bug today :)

**Pull request merged**